### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by kaizipaul**
**In this PR, I made accessibility checks on my portfolio, and here's what I found:**
- Page title: changed to "Portfolio Home Page"
- Image text alternatives: made changes to the alt texts in the HTML file
- Text headings: no issues found
- Color contrast: no issues found
- Resize: styled the header to expand while zooming in text
- Interaction: no issues found
- Moving content: no issues found
- Multimedia: no issues found
- The basic structure of the page: no issues found

 **Here is where I made the changes:**
- https://github.com/kaizipaul/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R7
- https://github.com/kaizipaul/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R53-R73
- https://github.com/kaizipaul/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R87
- https://github.com/kaizipaul/portfolio-project/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R282-R304
- https://github.com/kaizipaul/portfolio-project/pull/8/files#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aR598-R611

**Thank you for reviewing my PR**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <title>Portfolio Home Page</title>
```
Comment (1GORDON) [inline index.html]: Great job team 🤝

- [x] I would suggest you use something more specific for your title such as `Kaizi Paul Portfolio`.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @kaizipaul & @Allenkeys ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly make sure your text does not overlap or disappear whenever it is enlarged
text cut-off


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Allenkeys) [reply index.html]: @1GORDON Please, why is this project submitted on by dashboard for review, when I did not do it by myself and I'm only a contributor? I hope this meets you well :grin:

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @kaizipaul 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights
 - Followed Gitflow ✔️
 - Good commit messages ✔️
 - Descriptive Pull request ✔️
 - Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**